x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/new_entity_flyout.cy.ts

```diff
@@ -67,6 +67,14 @@ describe(
         expandRiskInputsFlyoutPanel();
         cy.get(RISK_INPUT_PANEL_HEADER).should('exist');
       });
+
+      it('should show asset criticality in the risk input panel', () => {
+        waitForAlerts();
+        expandFirstAlertUserFlyout();
+        expandRiskInputsFlyoutPanel();
+        cy.get(ASSET_CRITICALITY_BADGE).should('exist');
```

Comment:
NIT: Below assertion is implicitly checking for the existence of the element, so this can be removed.